app/views/LibraryInfoPage.scala.html

```diff
@@ -29,7 +29,7 @@
 )
 
 @(library                : GitRepository,
-  repositoryModules      : Option[RepositoryModules],
+  metaInfo               : Seq[RepositoryModules],
```

Comment:
The name `metaInfo` is a bit ambiguous (especially when you see things like `metaInfo.headOption`), I'd be tempted to leave it as `repositoryModules` (even if there is a lack of double plural)?